app/src/androidTest/java/ch/epfl/qedit/edit/EditOverviewFragmentTest.java

```diff
@@ -22,6 +26,21 @@
     public final FragmentTestRule<?, EditOverviewFragment> testRule =
             FragmentTestRule.create(EditOverviewFragment.class);
 
+    @Rule
```

Comment:
How come that you had to add these lines when the tests themselves did not change? Maybe I'm missing something here.

Reply:
That's because EditOverviewFragment depends on the EditActivity itself. Each time it gave me a null pointer exception. This resolved it.